src/screens/FeedScreen/Footer.js

```diff
@@ -12,7 +12,7 @@ import MemoIcShare from '../../assets/icons/Ic_share';
 import MemoIcComment from '../../assets/icons/Ic_comment';
 import MemoIcArrowDownVoteOn from '../../assets/arrow/Ic_downvote_on';
 import MemoIcArrowUpvoteOn from '../../assets/arrow/Ic_upvote_on';
-import {COLORS} from '../../utils/theme';
+import {colors} from '../../utils/colors';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to use `colors` instead of `COLORS`. Ensure that the `colors` module exports the necessary color constants and that they are correctly referenced throughout the codebase.
```diff
- import {COLORS} from '../../utils/theme';
+ import {colors} from '../../utils/colors';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->